src/mobile/patches/react-native-keychain+3.0.0.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/node_modules/react-native-keychain/RNKeychainManager/RNKeychainManager.m b/node_modules/react-native-keychain/RNKeychainManager/RNKeychainManager.m
+index 381981b..80d1d9f 100644
+--- a/node_modules/react-native-keychain/RNKeychainManager/RNKeychainManager.m
++++ b/node_modules/react-native-keychain/RNKeychainManager/RNKeychainManager.m
```

Comment:
What's the purpose of this patch?

Reply:
It fixes a memory leak on iOS (https://github.com/oblador/react-native-keychain/pull/188). It was included in `react-native-keychain@3.1.0` but that caused issues on Android, so I'm just patching it for now until we can migrate to 3.1.0.